core/server/data/migrations/versions/5.0/2022-05-10-14-57-cleanup-invalid-posts-status.js

```diff
@@ -0,0 +1,15 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const affectedRows = await knex('posts')
```

Comment:
Should we have a log message before the migration starts so it's clear it failed on this migration if there's an error?

Reply:
Done 👍🏻 

I'll also update my other migration that's merged with the same: https://github.com/TryGhost/Ghost/blob/main/core/server/data/migrations/versions/5.0/2022-05-09-14-17-cleanup-invalid-users-status.js